LibraAPI/Controllers/UserController.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using DataContent.DAL.Interfaces;
+using DataContent.DAL.Repositories;
+using DataContent.DAL.UserExceptions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using ModelLibrary;
+using ModelLibrary.DataContexts;
+using Newtonsoft.Json;
+
+namespace LibraAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UserController : ControllerBase
+    {
+        private readonly IUserRepository _repository;
+
+        public UserController(IUserRepository repository)
+        {
+            _repository = repository;
+        }
+
+        // GET: api/User
+        [HttpGet("/login/{email}/{password}")]
+        public async Task<ActionResult<User>> Login(string email, string password)
+        {
+            User user;
+            try
+            {
+                user = await _repository.LoginAsync(email, password);
+            }
+            catch(UserNotFoundException e)
+            {
+                return NotFound(e.Message);
+            }
+            catch(WrongUserPasswordException e)
+            {
+                return NotFound(e.Message);
+            }
```

Comment:
if you would use one custom exception you there would be less code and it will become reusable

Reply:
already seen that you have fixed it. good job!

Reply:
Already done with previous fixes